sympy/simplify/_cse_diff.py

```diff
@@ -0,0 +1,244 @@
+"""Module for differentiation using CSE."""
+
+from sympy import cse, Matrix, SparseMatrix, Derivative, MatrixBase
+from collections import Counter
+import re
+
+
+def _postprocess(repl, reduced):
+    """
+    Postprocess the CSE output to remove any CSE replacement symbols from the arguments
+    of Derivative terms.
+    """
```

Comment:
Best to make complete docstrings here now, so we know what the input/output exactly should be.